domains/shlok.json

```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Portfolio Website",
+  "owner": {
+    "username": "Shlok Madhekar",
```

Comment:
This needs to be your GitHub username.

@andrewstech this is why the checks were failing.

Reply:
> This needs to be your GitHub username.
> 
> @andrewstech this is why the checks were failing.

True

Reply:
I just fixed it. Lmk if there is any other issue